utils/test_harness/tools/src/test_harness_metric.cpp

```diff
@@ -468,11 +474,15 @@ metric_query_create(zet_metric_query_pool_handle_t metricQueryPoolHandle) {
 }
 
 void destroy_metric_query(zet_metric_query_handle_t metricQueryHandle) {
-  EXPECT_EQ(ZE_RESULT_SUCCESS, zetMetricQueryDestroy(metricQueryHandle));
+  if (metricQueryHandle != nullptr) {
```

Comment:
not convinced of this change: If the test has a bug, this will hide it. Is there a way to  abort the test if the helper finds the handle is nullptr ?

Reply:
same for all the ones below

Reply:
With ASSERT(), Resolved, Thank you.